boudicca.events/eventcollectors/src/main/kotlin/events/boudicca/eventcollector/LocalEventCollectorDebuggerApp.kt

```diff
@@ -0,0 +1,56 @@
+package events.boudicca.eventcollector
+
+import base.boudicca.api.eventcollector.configuration.EventCollectorsConfigurationProperties
+import base.boudicca.api.eventcollector.debugger.EventCollectorDebugger
+import base.boudicca.api.eventcollector.runner.buildRunnerFor
+import base.boudicca.fetcher.FileBackedFetcherCache
+import events.boudicca.eventcollector.collectors.ZuckerfabrikCollector
+import org.springframework.boot.CommandLineRunner
+import org.springframework.boot.WebApplicationType
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.builder.SpringApplicationBuilder
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Profile
+import org.springframework.scheduling.annotation.EnableScheduling
+import java.io.File
+
+@Profile("debug")
+@SpringBootApplication
+@EnableScheduling
```

Comment:
pretty sure scheduling is not needed here? since we do not actually want it to run multiple times in debug mode